ll-property-images.html

```diff
@@ -149,14 +272,23 @@
       _itemToDelete = e.detail.imgId;
       this.$.confirmation.open();
       this.$.confirmation.setAttribute('img-id', e.detail.imgId);
+      dialogClosedEvent = 'deleteImage';
     },
     /**
-     * Fired when it received the ll-property-image-default event
+     * Fired when it receives the ll-property-image-default event
      * Will call the manager to set the image as the default property image.
      * @private
      */
-    _defaultImageChange: function(e) {
-      this.$.manager.setAsDefaultImage(e.detail);
+    _defaultImageChange: function(image) {
+      this.$.confirmDefault.open();
+      this.$.confirmDefault.setAttribute('img-id', image.detail.imgId);
```

Comment:
It might be nice to refactor these to a single object, so you only call the set/getAttribute once rather than 7 times